native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/xslt-native/resource-config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resources":{
```

Comment:
Lets format this file

Reply:
updated in [c1d9a24](https://github.com/ballerina-platform/module-ballerina-xslt/pull/428/commits/c1d9a247da51437c8331cf0a6d78b5d336597a25)